src/table/components/TableHeadWrapper.jsx

```diff
@@ -100,6 +103,7 @@ function TableHeadWrapper({
           );
         })}
       </TableRow>
+      {showTotals(layout) && showTotalsAtTop(layout) && <TableTotals {...props} />}
```

Comment:
there is a nice trick you can do to avoid the props drilling here. Bit hard to explain I can show you tomorrow